### ISSUE TITLE
Query related to  output of mirdeep2.pl script of mirdeep2 

### ISSUE THREAD

**Issue by 99ARITRA** [rajewsky-lab/mirdeep2#114]
I recently ran the mirdeep2.pl script for identifying novel miRNA sequences in plants. I got the following message in the log file that was generated after the script finished running:

>parsing input file finished
>checking Rfam for hits to precursors
>Build bowtie index of Rfam entries

>Mapping mature, star and loop sequences against index
>Setting the index via positional argument will be deprecated in a future release. Please use -x option instead.
> reads processed: 6347
>reads with at least one alignment: 8 (0.13%)
>reads that failed to align: 6339 (99.87%)
>Reported 17 alignments

This message is usually displayed after running Bowtie and usually a SAM file and an aligned and/or unaligned fasta file is generated as an output and mirdeep2.pl uses bowtie . But here none of these files generated. 

Does the mirdeep2.pl script not generate these files ?

**Comment by Drmirdeep** [comment]
Did you try to run the tutorial ? intermediate files are usually put into the tmp folder

**Comment by 99ARITRA** [comment]
Yes I did run the tutorial files of _C. elegans (reported 0 alignments) and checked the tmp folder in mirdeep_runs. It contained files related to precursors ,precursors_for_randfold, an .mrd, and some .arf files. 

I also checked the files in the tmp folder of my own experimental run. They are the same files. However, I could not find files related to bowtie.

Also I did find another folder called mirna_results_21_06_2023_t_01_52_23 which contained the .fa and .bed files of novel mature, loops and star mirnas which were created in the final step of mirdeep2.pl. I am not clear if these files are the results of bowtie.

Also for your information this is the command line I used to run mirdeep2.pl:
`miRDeep2.pl collapsed_reads.fa chickpea_genome_mod.fa novel_prediction_mirna.arf none none none 2>report.log`
